frontend/libs/health-department/feature-case-detail/src/lib/edit/edit.component.ts

```diff
@@ -38,41 +29,56 @@ export interface CaseDetailResult {
   templateUrl: './edit.component.html',
   styleUrls: ['./edit.component.scss'],
 })
-export class EditComponent implements OnInit, OnChanges, OnDestroy {
+export class EditComponent implements OnInit, OnDestroy {
   private subs: SubSink = new SubSink();
-  ClientType = ClientType;
   today = new Date();
   errorGenerator = ValidationErrorGenerator;
   selectableIndexCases: CaseSearchItem[] = [];
+  type$$ = new BehaviorSubject<CaseType>(CaseType.Index);
 
   get isIndexCase() {
-    return this.type === ClientType.Index;
+    return this.type$$.value === CaseType.Index;
   }
 
-  @Input()
-  caseDetail: CaseDetailDto;
-  @Input()
-  type: ClientType;
-  @Input() loading: boolean;
-  @Output()
-  changedToIndex = new EventEmitter<boolean>();
+  caseDetail$: Observable<CaseDto>;
+  loading$: Observable<boolean>;
 
   formGroup: FormGroup;
   @ViewChild('editForm') editFormElement: NgForm;
 
-  @Output()
-  submittedValues: Subject<CaseDetailResult> = new Subject<CaseDetailResult>();
-
   constructor(
     private dialog: MatDialog,
     private snackbarService: SnackbarService,
-    public indexCaseService: IndexCaseService
+    public indexCaseService: IndexCaseService,
+    private route: ActivatedRoute,
+    private router: Router,
+    private entityService: CaseEntityService
   ) {}
 
   ngOnInit(): void {
     this.createFormGroup();
 
-    this.updateFormGroup(this.caseDetail);
+    this.route.parent.paramMap.pipe(map((paramMap) => paramMap.get('type'))).subscribe((type) => {
```

Comment:
Wenn du hier this.route.snapshot.parent.paramMap verwendest, kannst du dir das subscribe sparen. Andernfalls bitte die subscription zu subsink hinzufügen und bei onDestroy unsubscriben

Reply:
Muss type$$ wirklich ein BehaviorSubject sein oder kann es auch einfach eine normale Property sein?